Resources/Prototypes/Entities/Clothing/Shoes/misc.yml

```diff
@@ -181,3 +181,29 @@
     price: 75
   - type: Tag
     tags: [ ]
+   
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoesBootsJump
+  name: jump boots
+  description: High-tech boots used by explorers of new worlds to avoid unwanted obstacles.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/jumpboots.rsi
+    layers:
+    - state: icon
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/jumpboots.rsi
+  - type: JumpAbility
+    jumpDistance: 4
+    jumpSound: /Audio/Effects/stealthoff.ogg
+  - type: ActionGrant
+    actions:
+    - ActionGravityJump 
+  - type: ItemActionGrant
+    actions:
+    - ActionGravityJump 
```

Comment:
this code is really weird because ItemActionGrant lets you define actions but they are never used. I'd say just remove it from the lower one and once someone fixes the buggy behavior it can be changed to the more logical configuration.